02/styles/global.css

```diff
@@ -1,26 +1,44 @@
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
 }
 
-.a, .b, .c {
-    border: 3px solid transparent;
-    min-height: 50px;
+.a,
+.b,
+.c {
+	border: 10px solid transparent;
+	min-height: 50px;
 
-    display: flex;
-    justify-content: center;
-    align-items: center;
+	display: flex;
+	justify-content: center;
+	align-items: center;
 }
 
 .a {
-    border-color: #ffffba
+	border-color: #ffffba;
+	height: 50vh;
 }
 
 .b {
-    border-color: #ffdfba;
+	border-color: #ffdfba;
+	height: 50vh;
 }
 
 .c {
-    border-color: #baffc9;
-}
\ No newline at end of file
+	border-color: #baffc9;
+	height: 50vh;
+}
```

Comment:
Na mobile miało być:
A
B
C

A nie tak jak jest teraz tj.

A
C
B

Jeśli to przeoczenie to postaraj się bardziej uważać. Na rekrutacji zazwyczaj są bardziej podchwytliwe zadania :P